R/utils.R

```diff
@@ -192,10 +193,23 @@ render_dependencies <- function(dependencies, local = TRUE, prefix=NULL) {
                                                                        dep[["stylesheet"]], 
                                                                        sep="/"))
     } else if ("stylesheet" %in% names(dep) & src == "file") {
+      dep[["stylesheet"]] <- paste0(path_prefix,
+                                "_dash-component-suites/",
+                                dep$name,
+                                "/",
+                                basename(dep[["stylesheet"]]),
+                                sprintf("?v=%s&m=%s", dep$version, modified))
+      
       if (!(is.null(dep$version))) {
-        html <- sprintf("<link href=\"%s?v=%s\" rel=\"stylesheet\" />", file.path(dep[["src"]][["file"]],
-                                                                                  dep[["stylesheet"]]),
-                        dep$version)        
+        if(!is.null(dep$package)) {
+          html <- sprintf("<link href=\"%s?v=%s\" rel=\"stylesheet\" />", file.path(dep[["stylesheet"]]),
+                          dep$version)        
```

Comment:
why is `dep$version` appearing both here and above where you define `dep[["stylesheet"]]`?

Reply:
:facepalm: fixed in d9d66c13062da253b0f77bb131126f64f12e8516